martenbuild.cs

```diff
@@ -78,9 +89,42 @@ private static void Main(string[] args)
             Target("pack", DependsOn("compile"), ForEach("./src/Marten", "./src/Marten.CommandLine", "./src/Marten.NodaTime"), project =>
                 Run("dotnet", $"pack {project} -o ./../../artifacts --configuration Release"));
 
+            Target("init-db", () =>
```

Comment:
Nice, I like that idea 👍

Reply:
I use that for something similar at work, but w/ Sql Server